dataland-frontend/tests/component/components/pages/LandingPage.cy.ts

```diff
@@ -0,0 +1,126 @@
+   * Gets the top bar element
+   * @returns the top bar element
+   */
+  function getTopBar(): Cypress.Chainable {
+    return cy.get("header");
+  }
+  checkImage("Dataland banner logo", "gfx_logo_dataland_orange_S.svg", getTopBar());
+  checkButton("signup_dataland_button", "Sign Up", getTopBar());
+  checkLinkByContent("Login", getTopBar());
+}
+
+/**
+ * Validates the elements of the intro section
+ */
+function validateIntroSection(): void {
+  checkImage("Liberate Data -  Empower Autonomy. Dataland, the Open ESG Data Platform.", "gfx_logo_d_orange_S.svg");
+  cy.get("h1").should("contain.text", "Liberate Data");
+  cy.get("h1").should("contain.text", "Empower Autonomy");
+}
+
+/**
+ * Validates the images of the brands trusting in dataland
+ */
+function validateBrandsSection(): void {
+  checkImage("Brand 1", "img_chom_capital.png");
+  checkImage("Brand 2", "img_envoria.png");
+  checkImage("Brand 3", "img_fuerstl_bank.png");
+  checkImage("Brand 4", "img_ampega.png");
+  checkImage("Brand 5", "img_metzler.png");
+  checkImage("Brand 6", "img_deka.png");
+}
+
+/**
+ * Check the new footer
+ */
+function checkNewFooter(): void {
+  /**
+   * Gets the footer element
+   * @returns the footer element
+   */
+  function getFooter(): Cypress.Chainable {
+    return cy.get("footer");
+  }
+  getFooter().should("exist");
+  checkImage("Copyright ©   Dataland", "gfx_logo_dataland_orange_S.svg", getFooter());
+  cy.get(".footer__copyright").should("contain.text", "Copyright © 2023 Dataland");
+
+  checkLinkByTarget("/legal", "Legal", getFooter());
+  checkLinkByTarget("/imprint", "Imprint", getFooter());
+  checkLinkByTarget("/dataprivacy", "Data Privacy", getFooter());
+}
+
+/**
+ * Validates that the slide show on "Quotes" works as expected
+ */
+function validateQuotesSlides(): void {
+  /**
+   * Gets the "Quotes" element
+   * @returns the "Quotes" element
+   */
+  function getSlidesWrapper(): Cypress.Chainable {
+    return cy.get("section.quotes");
+  }
+
+  getSlidesWrapper().find(".quotes__slides").should("have.css", "transform", "none");
+  getSlidesWrapper().find("button[aria-label='Next slide']").click();
+  getSlidesWrapper().find(".quotes__slides").should("have.css", "transform", "matrix(1, 0, 0, 1, -440, 0)");
+  getSlidesWrapper().find("button[aria-label='Next slide']").click();
+  getSlidesWrapper().find(".quotes__slides").should("have.css", "transform", "matrix(1, 0, 0, 1, -440, 0)");
+  getSlidesWrapper().find("button[aria-label='Previous slide']").click();
+  getSlidesWrapper().find(".quotes__slides").should("have.css", "transform", "matrix(1, 0, 0, 1, 0, 0)");
+  getSlidesWrapper().find("button[aria-label='Previous slide']").click();
+  getSlidesWrapper().find(".quotes__slides").should("have.css", "transform", "matrix(1, 0, 0, 1, 440, 0)");
+  getSlidesWrapper().find("button[aria-label='Previous slide']").click();
+  getSlidesWrapper().find(".quotes__slides").should("have.css", "transform", "matrix(1, 0, 0, 1, 440, 0)");
```

Comment:
Can you please explain this to me in a call? I think it's faster than me finding out on my own.